CHANGELOG.md

```diff
@@ -1,11 +1,31 @@
 # Braintree Android SDK Release Notes
 
-## Unreleased
+## unreleased
 * Adds `invalidateClientToken` method to `BraintreeClient`, which invalidates the cached client
 token authorization provided by the `ClientTokenProvider`
 * Add `isExplicitCancelation` parameter to `UserCanceledException`
 * Trim tokenization key and client token before parsing
-
+* PayPalNativeCheckout
+  * Adding in [PayPalNativeCheckout] module to use the native checkout for PayPal
```

Comment:
```suggestion
  * Add new module to use Native Checkout for PayPal transactions
```

Reply:
Nitpick. We rarely add new modules so we don't have a set practice for announcing new features via CHANGELOG.